<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_input">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс поискогового результата</string>
    </property>
   </widget>
   <widget class="QLabel" name="address_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Полный адрес (выполните поиск)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="postal_box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>